lib/launcher.js

```diff
@@ -160,7 +160,7 @@ class AppiumLauncher {
         }
         return ret;
     }
-    
+ // eslint-disable-line    no-trailing-spaces 
```

Comment:
Why is this rule disabled?

Reply:
I was not able to fix this, and its probably due to windows. I am working on mac now to solve other problems

Reply:
Fixed